code/espurna/mqtt.ino

```diff
@@ -17,41 +18,38 @@ Copyright (C) 2016-2019 by Xose Pérez <xose dot perez at gmail dot com>
 #include <Ticker.h>
 #include <TimeLib.h>
 
-#if MQTT_LIBRARY == MQTT_ASYNC // AsyncMqttClient
-    #include <AsyncMqttClient.h>
+#include "libs/SecureClientHelpers.h"
+
+#if MQTT_LIBRARY == MQTT_LIBRARY_ASYNCMQTTCLIENT
+
     AsyncMqttClient _mqtt;
-#else // Arduino-MQTT or PubSubClient
-    WiFiClient _mqtt_client;
-    bool _mqtt_connected = false;
 
-    #include "WiFiClientSecure.h"
+#else // MQTT_LIBRARY_ARDUINOMQTT / MQTT_LIBRARY_PUBSUBCLIENT
 
-    #if SECURE_CLIENT == SECURE_CLIENT_AXTLS
-        using namespace axTLS;
-    #elif SECURE_CLIENT == SECURE_CLIENT_BEARSSL
-        using namespace BearSSL;
-        BearSSL::X509List *_ca_list = nullptr;
-    #endif
+    WiFiClient _mqtt_client;
 
-    WiFiClientSecure _mqtt_client_secure;
+#if SECURE_CLIENT != SECURE_CLIENT_NONE
+    SecureClient* _mqtt_client_secure = nullptr;
+#endif
 
-    #if MQTT_LIBRARY == MQTT_ARDUINO // Using Arduino-MQTT
-        #include <MQTTClient.h>
-        #ifdef MQTT_MAX_PACKET_SIZE
+#if MQTT_LIBRARY == MQTT_LIBRARY_ARDUINOMQTT
+#ifdef MQTT_MAX_PACKET_SIZE
         MQTTClient _mqtt(MQTT_MAX_PACKET_SIZE);
```

Comment:
And what is up with this btw. Does platformio build_flags need comment / bigger value for it? idk if json works at all with the default 400 bytes buffer